ts/features/idpay/initiative/configuration/screens/InstrumentsEnrollmentScreen.tsx

```diff
@@ -208,6 +199,18 @@ const InstrumentsEnrollmentScreen = () => {
     );
   };
 
+  const handleInstrumentValueChange = (wallet: Wallet) => (value: boolean) => {
+    if (value) {
+      setStagedInstrument(wallet);
+    } else {
+      const instrument = initiativeInstrumentsByIdWallet[wallet.idWallet];
```

Comment:
`instrument` here is possibly undefined.

I've noticed that rapidly tapping an unenrolled instrument's switch, so that it unenrolls before even enrolling or showing a bottomSheet, causes the screen to crash since the machine is trying to check for an undefined value's entries.
could you look into it?

Reply:
Should be resolved in https://github.com/pagopa/io-app/pull/4471/commits/966ccb2be87febe12c7f178e33298ceb65c92e93